src/content/docs/en/developers/ethereum-and-scroll-differences.mdx

```diff
@@ -33,7 +33,7 @@ For open-source contributors and infrastructure builders, please contact our tea
 
 ## EVM Precompiles
 
-The `SHA2-256` (address `0x2`), `RIPEMD-160` (address `0x3`), and `blake2f` (address `0x9`) precompiles are currently not supported. Calls to these precompiled contracts will revert. We plan to enable these three precompiles in a future hard fork.
+The `RIPEMD-160` (address `0x3`) and `blake2f` (address `0x9`) precompiles are currently not supported. The `SHA2-256` (address `0x2`) is currently supported on Scroll Sepolia, but will soon be supported on Scroll Mainnet. Calls to unsupported precompiled contracts will revert. We plan to enable these precompiles in future hard forks.
```

Comment:
Can we link to the upgrades page here? Maybe for "soon be supported on Scroll Mainnet"?